AutoHeuristic: script to generate data for mm

AlnisM opened the pull request:
This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #132088
* #131714
* #131710
* #131705
* __->__ #131617
* #131616
* #131615

AlnisM committed:
AutoHeuristic: script to generate data for mm

[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

eellison commented on torchgen/autoheuristic/mm/gen_data_mm.py:
```diff
@@ -0,0 +1,119 @@
+# mypy: ignore-errors
```
add docstring ? remove mypy: ignore-errors ?

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM committed:
Update on "AutoHeuristic: script to generate data for mm"

This PR introduces a script that can be used to generate training data for tuned_mm in order to learn a heuristic with AutoHeuristic.




[ghstack-poisoned]

AlnisM commented:
@pytorchbot merge
